.github/workflows/continous-integration-workflow.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       working-directory: .
       run: | 
         docker build --target e2etest --tag todo-app:e2etest .
-        docker run --env SECRET_KEY=${{ secrets.SECRET_KEY }} --env TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} --env TRELLO_TOKEN=${{ secrets.TRELLO_TOKEN }} --env TRELLO_BOARD_NAME=${{ secrets.TRELLO_BOARD_NAME }} todo-app:e2etest      
+        docker run --env SECRET_KEY=${{ secrets.SECRET_KEY }} --env MONGO_CONNECTION_STRING=${{ secrets.MONGO_CONNECTION_STRING }} --env DBNAME=${{ secrets.DBNAME }} --env COLLECTIONAME=${{ secrets.COLLECTIONAME }} todo-app:e2etest
```

Comment:
No reason to pass in DBNAME since the e2e test itself sets the DBNAME to 'E2E_Test_DB'

Reply:
ok